charts/k8s-monitoring/docs/Structure.md

```diff
@@ -0,0 +1,82 @@
+# Structure
+
+The Kubernetes Monitoring Helm chart contains many software packages and builds a comprehensive set of configuration and
+secrets for those packages.
+This document aims to describe and explain the structure of the Helm chart.
+
+![Kubernetes Monitoring inside of a Cluster](https://grafana.com/media/docs/grafana-cloud/k8s/Helm-chart-agent-diagram.png)
+
+## Software Deployed
+
+This Helm chart deploys several packages to generate and capture the telemetry data on the cluster. This list
+corresponds to the list of dependencies in this chart's Chart.yaml file. For each package, there is an associated
+section inside the Helm chart's values.yaml file that controls how it is configured.
+
+| Name                                                                                   | Type                      | Associated values             | Description                                                                                                                                                                                                          | 
+|----------------------------------------------------------------------------------------|---------------------------|-------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Grafana Agent](https://grafana.com/oss/agent/)                                        | StatefulSet               | `grafana-agent`               | The Grafana Agent that is responsible for scraping metrics, and accepting metrics, logs, and traces via receivers                                                                                                    |
+| Grafana Agent for Logs                                                                 | DaemonSet                 | `grafana-agent-logs`          | The Grafana Agent that gathers pod logs. By default, it uses HostPath volume mounts to read pod log files directly from the nodes. It can alternatively get logs via the API server and be deployed as a Deployment. |
```

Comment:
```suggestion
| Grafana Agent for Logs                                                                 | DaemonSet                 | `grafana-agent-logs`          | The Grafana Agent that gathers Pod logs. By default, it uses HostPath volume mounts to read Pod log files directly from the nodes. It can alternatively get logs via the API server and be deployed as a Deployment. |
```